src/css/main.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../node_modules/reset.css/reset.css';
```

Comment:
Rename this file to `css` or change `postcss` in the all affected components from this PR to `scss`

Reply:
it should be in separated task

Reply:
Rename this file to css

Reply:
why? `import` will not working